data/json/recipes/recipe_others.json

```diff
@@ -1170,6 +1170,20 @@
     "tools": [ [ [ "welder", 50 ], [ "welder_crude", 75 ], [ "toolset", 75 ], [ "oxy_torch", 10 ] ] ],
     "components": [ [ [ "wire", 6 ], [ "basket", 1 ] ], [ [ "scrap", 8 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "bike_basket",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_PARTS",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": 5000,
+    "reversible": true,
+    "autolearn": true,
+    "qualities": [ { "id": "GLARE", "level": 2 } ],
```

Comment:
You need `SAW_M` quality or bolt cutter item to cut wires.

Reply:
Is there any way to make it a OR between a quality and a specific tool? Or would I need to make a different quality for bolt cutting if I wanted to make it `saw_m or bolt cutter`.

Reply:
I guess it is not possible to define tool OR quality yet.

Reply:
I was adding a bolt cutting quality, but it doesn't seem to be possible to do a OR with qualities either, so I won't push that commit up. At `src/requirements.cpp`

```
        } else {
            // tool qualities don't normally use a list of alternatives
            // each quality is mandatory.
            objs.push_back( std::vector<T>( 1 ) );
            objs.back().back().load( jsarr );
}
```

Reply:
It seems there are these options:

- use single quality in recipe (i.e. `SAW_M`);
- use single tool in recipe (i.e. `bolt_cutter`);
- add new quality (e.g. `CUT_WIRE`), add it to `boltcutters` and `hacksaw` tools and use new quality in recipe;
- add new requirement which includes both `boltcutters` and `hacksaw` in tools and use it in recipe:

```JSON
  {
    "id": "cutting_wire_standard",
    "type": "requirement",
    "//": "Cutting wires",
    "tools": [
      [ [ "hacksaw", -1 ], [ "multitool", -1 ],  [ "boltcutters", -1 ], [ "toolset", -1 ] ]
    ]
  },
```

Reply:
Thanks a bunch! Learned a fair bit that I didn't know..

I went with the last option, seems better given the qualities being mandatory. There are quite a few recipes that is cutting wires, so I'll take a look at porting them in a future PR.